doc/classes/RandomNumberGenerator.xml

```diff
@@ -4,6 +4,7 @@
 		A class for generation pseudo-random numbers.
 	</brief_description>
 	<description>
+		RandomNumberGenerator is a class for generation pseudo-random numbers. It currently uses PCG32. The underlying algorithm is an implementation detail. As a result, it should not be depended upon for reproducible random streams across Godot versions.
```

Comment:
Should be: `for generating pseudo-random numbers` or `for pseudo-random numbers generation`.

Reply:
True... I just copied `brief_description` without even reading it. Fixed both.